src/ServicePulse.Host/vue/src/composables/formatter.ts

```diff
@@ -11,7 +11,7 @@ export interface ValueWithUnit {
 }
 
 export function useFormatTime(value?: number): ValueWithUnit {
-  const time = { value: "0", unit: "" };
+  const time = { value: "0", unit: "ms" };
```

Comment:
I wonder why none of the if statements were working for this condition. https://github.com/Particular/ServicePulse/blob/772e86e803087e1fa30fa46577bddfdc77acc519/src/ServicePulse.Host/vue/src/composables/formatter.ts#L29-L31

Reply:
it isn't hitting this else statement because of the surrounding
``` cs
 if (value) {
   ...
```